src/components/molecules/CardInfo.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const CardInfo = () => {
```

Comment:
It wil be better if we configure the props for send the necessary information.
```suggestion
const CardInfo = ({title, description}) => {
```

Reply:
Done!